package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "rollup -c",
     "prepublishOnly": "npm run build",
+    "prepare": "npm run build",
```

Comment:
I think that it's better to remove this line, since you don't want to force the user to build right away the package on install.
```suggestion
```

Reply:
I added this to use this package straight from my GH, so yea we can remove it